packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-transfer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useMemo } from 'react';
+import { Movement } from '@carbon/react/icons';
+import styles from '../ward-patient-card.scss';
+import { useTranslation } from 'react-i18next';
+import { type WardPatient } from '../../types';
+
+export interface WardPatientTransferProps {
+  wardPatient: WardPatient;
+}
+
+const WardPatientTransfer: React.FC<WardPatientTransferProps> = ({ wardPatient }) => {
+  const { t } = useTranslation();
+
+  const { dispositionType } = wardPatient?.inpatientRequest;
+  const message = useMemo(() => {
+    if (dispositionType === 'TRANSFER') {
+      return t('transferToDeliveryWard', 'Transfer to Delivery ward');
+    }
+    if (dispositionType === 'DISCHARGE') {
+      return t('transferToDeliveryWard', 'Transfer to Delivery ward');
+    }
+    return '';
+  }, [dispositionType]);
+
+  if (dispositionType === 'ADMIT') return null;
```

Comment:
Just to be safe if we start adding other dispositions, let's make this display null for any dispositionType (or null dispositionType) other than TRANSFER and DISCHARGE.